html/editing/editing-0/contenteditable/contentEditable-shortcuts.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Editing: manual test for keyboard shortcuts with contenteditable=plaintext-only</title>
+    <link rel="help" href="https://html.spec.whatwg.org/multipage/#contenteditable"/>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <script src="/resources/testdriver.js"></script>
+    <script src="/resources/testdriver-vendor.js"></script>
+    <script src="/resources/testdriver-actions.js"></script>
+
+    <div id="contentEditablePlaintextOnly" contenteditable="plaintext-only">plaintext-only</div>
+</head>
+<body>
+<script>
+    var kControl = "\uE009";
+    var kMeta = "\uE03D";
+    var kB = "B";
+
+    promise_test(async () => {
+        var contentEditablePlaintextOnly = document.getElementById('contentEditablePlaintextOnly');
+
+        var selection = document.getSelection();
+        selection.selectAllChildren(contentEditablePlaintextOnly);
+
+        await new test_driver.Actions()
+            .keyDown(kControl)
+            .keyDown(kB)
+            .keyUp(kB)
+            .keyUp(kControl)
+            .send();
+
+        assert_equals(document.queryCommandState('bold'), false);
```

Comment:
Comments appreciated. Is the intention the `queryCommandState('bold')` should be `false` because the text is already bold by the shortcut?

Reply:
It's because for plaintext-only, bold shouldn't work. I'll clarify this in the test name.